src/components/Step.vue

```diff
@@ -248,24 +313,73 @@ export default {
 }
 </script>
 
-<style lang="scss" scoped>
+
+<style lang="scss">
```

Comment:
We should decide how we want to write CSS. Should we use SASS or not. Naming conventions for CSS classes (like SuitCSS), use Tailwind or Bulma etc.
Otherwise everything will soon be a total mess.

Reply:
I agree. 
I vote for sass 🙂